src/index.html

```diff
@@ -6,13 +6,76 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="./styles/card.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/stars.scss"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="../src/images/imac.jpeg"
+        alt="imac"
+        class="card__product-image"
+      />
+
+      <h3 class="card__product-name">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </h3>
```

Comment:
The 'h3' tag is not the best choice for the product name. It's better to use 'h1' for the main title of the page. Also, you should avoid using 'br' tag to create a new line. It's better to use CSS for this purpose.